Multisig/Logic/Models/Address.swift

```diff
@@ -43,3 +43,28 @@ extension Address {
         try? self.init(hex: value, eip55: false)
     }
 }
+
+extension Address {
+
+    init(_ value: String, _ isERC681: Bool) throws {
```

Comment:
please make the 2nd argument named, because it is not clear what "true" means without the isERC681 label